ffs/api/api.go

```diff
@@ -150,18 +194,26 @@ func (i *API) Info(ctx context.Context) (InstanceInfo, error) {
 	if err != nil {
 		return InstanceInfo{}, fmt.Errorf("getting pins from instance: %s", err)
 	}
-	balance, err := i.wm.Balance(ctx, i.cfg.WalletAddr)
-	if err != nil {
-		return InstanceInfo{}, fmt.Errorf("getting balance of %s: %s", i.cfg.WalletAddr, err)
+
+	wallet := make([]BalanceInfo, len(i.cfg.Addrs))
+	j := 0
+	for _, addr := range i.cfg.Addrs {
```

Comment:
You can use `for j, addr := ..`

Reply:
Oh, I thought `j` would be the `string` key of the map in that case.

Reply:
(not an `int` index since we're ranging over a `map`)

Reply:
Oh, thought was a slice. Dismiss!

Reply:
anyway, works better just using `append`